jacoco.gradle

```diff
@@ -0,0 +1,46 @@
+apply plugin: 'jacoco'
```

Comment:
Can we move this to a subfolder, maybe config ?

Help keep our base directory as straightforward as possible